Misc/NEWS.d/next/Library/2021-08-10-00-05-53.bpo-44859.9e9_3V.rst

```diff
@@ -0,0 +1,7 @@
+Raise more accurate exceptions in :mod:`sqlite3`.
+
+* Raise InterfaceError iso. ProgrammingError for SQLITE_MISUSE
+* Don't overwrite BufferError with ValueError when converting to BLOB
+* Raise ProgrammingError iso. Warning if user tries to execute() more than one
+  statement
+* Raise DataError iso. ValueError if query contains NULL characters
```

Comment:
```suggestion
* Raise ProgrammingError instead of Warning if user tries to execute() more than one
  statement
* Raise DataError instead of ValueError if query contains NULL characters
```